helm-chart/sefaria-project/templates/rollout/linker.yaml

```diff
@@ -0,0 +1,191 @@
+    blueGreen:
+      activeService: linker-{{ .Values.deployEnv }}
+      autoPromotionEnabled: true
+      prePromotionAnalysis:
+        templates:
+          # - templateName: rollout-readiness-{{ .Values.deployEnv }}
+          - templateName: rollout-ready-{{ .Values.deployEnv }}
+        args:
+          - name: healthcheck-hostname
+            value: "nginx-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
+  selector:
+    matchLabels:
+      app: linker-{{ .Values.deployEnv }}
+      # releaseRevision: "{{ .Release.Revision }}"
+  revisionHistoryLimit: 1
+  replicas: {{ .Values.linker.replicaCount }}
+  progressDeadlineSeconds: 1200
+  template:
+    metadata:
+      labels:
+        app: linker-{{ .Values.deployEnv }}
+        tier: application
+        deployEnv: "{{ .Values.deployEnv }}"
+        stackRole: django
+        releaseRevision: "{{ .Release.Revision }}"
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - mongo
+            topologyKey: kubernetes.io/hostname
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - "linker-{{ .Values.deployEnv }}"
+            topologyKey: kubernetes.io/hostname
+        nodeAffinity:
+          {{- include "sefaria.nodeAffinities" . | nindent 10 }}
+      containers:
+      - name: linker
+        image: "{{ .Values.linker.containerImage.imageRegistry }}:{{ .Values.linker.containerImage.tag }}"
+        imagePullPolicy: Always
+        args: [ "python manage.py migrate && gunicorn sefaria.wsgi --access-logfile - --error-logfile - --timeout 300 --threads {{ .Values.linker.resources.web.gunicornThreadCount }} --worker-tmp-dir /dev/shm -b 0.0.0.0:80" ]
+        env:
+          # WEB_CONCURRENCY is used for determining the number of server workers
+          - name: WEB_CONCURRENCY
+            value: "{{ .Values.linker.resources.web.gunicornWorkerCount }}"
+          - name: randomstringtoforceredeployments
+            value: {{ randAlphaNum 8 }}
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /app/logging-secret.json
+          - name: ENV_NAME
+            value: "{{ .Values.deployEnv }}"
+          - name: STACK_COMPONENT
+            value: web
+          - name: REDIS_HOST
+            value: "redis-{{ .Values.deployEnv }}"
+          - name: NODEJS_HOST
+            value: "node-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
+          - name: VARNISH_HOST
+            value: "varnish-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
+          - name: HELM_REVISION
+            value: "{{ .Release.Revision }}"
+          - name: ENABLE_LINKER
+            value: "True"
+          - name: DISABLE_AUTOCOMPLETER
+            value: "True"
```

Comment:
From what I understand, `local-settings-file.yaml` will run `bool("True")` which will be `True`. but `bool("False")` is also `True`. So not currently an issue but confusing for the future if someone changes this to `"False"`.

Reply:
That was not my intention.  Do you know a good way to handle this?  Doing a string compare instead of a parse seems a bit clunky but I'm not sure there's a better option?

Reply:
I would use `"0"` and `"1"` and then in then in the reading on the envvars use `bool(int(os.getenv(...)))`. It's not the prettiest but works

Reply:
I see there are some other places where we have used a string compare - for consistency I will do so here too